app/src/App.vue

```diff
@@ -28,8 +26,13 @@ export default {
   },
   methods: {
     swapComponent: function(component, payload) {
+      const el = document.body;
+
       if (component.valueOf() === "ConfViewer") {
         this.currentComponentPayload = this.events;
+        el.classList.add('conference');
```

Comment:
https://forum.vuejs.org/t/when-is-it-ok-to-directly-manipulate-the-dom/45818

Reply:
I am no expert in vue and this was the quickest solution I could come up with, you are welcome to do it another way if you have a better solution, but for now this works and is all I need